tagstudio/tests/core/test_tags.py

```diff
@@ -1,18 +1,161 @@
+import sys
+import os
+import pytest
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..')))
+
 from src.core.library import Tag
 
+@pytest.fixture
+def mock_library(mocker):
+    return mocker.Mock()
+
+@pytest.fixture
+def mock_subtag(mocker):
+    return mocker.Mock()
+
+@pytest.fixture
+def test_tag1():
+    return Tag(id=1, name="tag1 Name", shorthand="tag1", aliases=[], subtags_ids=[2], color="")
+
+@pytest.fixture
+def test_tag2():
+    return Tag(id=2, name="tag2 Name", shorthand="tag2", aliases=[], subtags_ids=[3], color="")
+
+@pytest.fixture
+def test_tag3():
+    return Tag(id=3, name="tag3 Name", shorthand="tag3", aliases=[], subtags_ids=[], color="")
+
+class TestTags():
```

Comment:
isnt this wrapping class superfluous?

Reply:
I dont understand how is a failing test (if I understand correctly) is related to the wrapping class. 
Pytest tests are usually done as a top-class functions. If a test is failing, add decorator `@pytest.mark.skip` or comment it out 🤔